src/MMLib.SwaggerForOcelot/Configuration/SwaggerForOcelotUIOptions.cs

```diff
@@ -32,5 +33,10 @@ public string PathToSwaggerGenerator
         /// <see cref="EndPointBasePath"/> + <see cref="SwaggerEndPointConfig.Version"/> + <see cref="SwaggerEndPointOptions.Key"/>
         /// </summary>
         public string DownstreamSwaggerEndPointBasePath { get; set; } = "/swagger/docs";
+        
+        /// <summary>
+        /// Headers to include when requesting a downstream swagger endpoint
```

Comment:
```suggestion
        /// Headers to include when requesting a downstream swagger endpoint.
```

Reply:
Hi,
I've updated the ReadME fixed the above error.